airflow/www/static/js/dag/details/gantt/Row.tsx

```diff
@@ -53,17 +53,21 @@ const Row = ({
   const instance = task.instances.find((ti) => ti.runId === runId);
   const isSelected = taskId === instance?.taskId;
   const hasQueuedDttm = !!instance?.queuedDttm;
+  const validQueuedDttm =
```

Comment:
There is a case where the task is queued to have queued_dttm and there is no start_date since the task has not started where hasQueuedDttm is true but validQueuedDttm won't be true since start_date is null and based on hasQueuedDttm the minimum bar for needs to be present.

Reply:
Ok, please ignore I see I combined them later and hasQueuedDttm is not used except for this check. Makes sense to combine them.